contracts/polygon/connectors/fluidity/main.sol

```diff
@@ -0,0 +1,108 @@
+pragma solidity ^0.8.6;
+pragma experimental ABIEncoderV2;
+
+import {Events} from "./events.sol";
+import {Helpers} "./helpers.sol";
+import { TokenInterface } from "../../common/interfaces.sol";
+
+abstract contract FluidityP1M2 is Events, Helpers {
+	function supply(
+		address token_,
+		uint256 amt,
+		uint256 getId,
+		uint256 setId
+	)
+		public
+		payable
```

Comment:
No need to make any of the functions payable, as we only deal with erc20

Reply:
done